tests/functional/ovncontroller_controller_test.go

```diff
@@ -534,15 +536,15 @@ var _ = Describe("OVNController controller", func() {
 
 			Eventually(func(g Gomega) {
 				g.Expect(th.GetConfigMap(externalCM).Data["ovsdb-config"]).Should(
-					ContainSubstring("ovn-remote: %s", externalSBEndpoint))
+					ContainSubstring("ovn-remote: %s", ExpectedExternalSBEndpoint))
 			}, timeout, interval).Should(Succeed())
 
 			newExternalSBEndpoint := "10.0.0.250"
```

Comment:
Mm... I don't know if this test make sense anymore. I guess the initial intention of this test was to cover the case where a pod was created, for some reason (not healty, reschedule) pod was destroyed and recreated (with a different IP), and this test was checking that the new value was propagated into the CM.

This is actually the main problem this PR resolves, that during restarting pods the CM won't be modified so external nodes (edpm nodes) will not notice the IP change.

So I think that we can remove this test, but on the other hand test passes fine as it is (If test is not deleted I think that we should change it's name).

PS: In case that we delete this test I could also delete the function "SetExternalEndpoint" as it is not needed on all other tests

Reply:
It's ok to remove it; indeed the stable address is the whole point of the PR.

Reply:
Removed.